public/demos/breakout.html

```diff
@@ -0,0 +1,31 @@
+<html>
+  <head>
+    <title>Daily.co Breakout Demo</title>
+    <script crossorigin src="https://unpkg.com/@daily-co/daily-js"></script>
+    <!--
+    This demo shows how you can implement a "breakout room" in a Daily Prebuilt room.
+    -->
+  </head>
+  <body onload="main()">
+    <div id="dailyContainer"></div>
+
+    <script type="module">
+      const urlParams = new URLSearchParams(window.location.search);
+      const domain = urlParams.get("domain") || "hush";
+      const room = urlParams.get("room") || "breakout1";
+
+      import daily from "../extensions/core.js";
+      import * as breakout from "../extensions/breakout.js";
+      window.main = async function main() {
+        window.call = daily.createFrame(dailyContainer, {
+          url: `https://${domain}.daily.co/${room}`,
+          dailyConfig: {
+            experimentalChromeVideoMuteLightOff: true,
+          },
+        });
+        await call.join();
+        breakout.connect({ domain, room });
```

Comment:
We could actually avoid even having to do this in the top-level app if we want; see my comment in breakout.js